### DISCUSSION TITLE
The usage of Chronos model.

### DISCUSSION THREAD

**Discussion by Riofd** [autogluon/autogluon#4211] [Q&A] [answered]
I read the docs of [Chronos model](https://auto.gluon.ai/stable/tutorials/timeseries/forecasting-chronos.html#:~:text=versions%20of%20AutoGluon.-,Does%20Chronos%20work%20with%20covariates%20or%20features%3F,-%C2%B6), and it says to support more features by `chronos_ensemble` **[The current iteration of Chronos does not support covariates or features, however we will provide this functionality in later versions. In the meanwhile, presets such as chronos_ensemble combine Chronos with models that do take advantage of features.]**, but I can't see any description of this method. Is there an example for this usage? Maybe it is not a new feature in `autogluon`?

**Comment by shchur** [comment, marked as answer]
Hi @Riofd, the `chronos_ensemble` preset trains Chronos[small] together with other models specified under `light_inference` in https://github.com/autogluon/autogluon/blob/master/timeseries/src/autogluon/timeseries/models/presets.py#L133, and then builds the WeightedEnsemble combining all of these models.

**Reply by Riofd** [reply]
Thank you for your reply. I noticed that Chronos does not support covariates, it means the `static_features` and `know_covariates` are invalid in train_df? So if I set `static_features` and `know_covariates`, other models in `chronos_ensembles` which support them can use there variates for training and prediction?

**Reply by shchur** [reply]
Yes, static features and known covariates will be ignored by the Chronos model, but they can be supported by other models in the ensemble.

**Reply by Riofd** [reply]
That's good. Moreover, may I ask that the Chronos model will support `static_features` and `known_covariates` in the future? Chronos is a good  zero-shot model, and I think it would have a much better performance if support them in more complex tasks.

**Reply by shchur** [reply]
Support for static features and covariates is something that we are currently exploring, but we don't yet have a specific date when this will be merged to AutoGluon.
